ros_gz_bridge/src/bridge_config.cpp

```diff
@@ -173,31 +174,30 @@ std::vector<BridgeConfig> readFromYaml(std::istream & in)
 std::vector<BridgeConfig> readFromYamlFile(const std::string & filename)
 {
   std::vector<BridgeConfig> ret;
-  std::ifstream in(filename);
+  resource_retriever::Retriever r;
```

Comment:
nit: single letter variable names violate our style guide.  Can you use a more descriptive variable?

Reply:
Renamed in [13f4008](https://github.com/gazebosim/ros_gz/pull/511/commits/13f40087d99b92c5bc4c8b157a951aad1efc0d9f)